.github/workflows/workflow.yml

```diff
@@ -8,105 +8,76 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
+  lint:
     runs-on: ubuntu-20.04
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+
+      - name: Install Poetry
+        run: pipx install poetry
```

Comment:
We could add `poetry config virtualenvs.in-project true` to make poetry install in `.venv` to cache this folder between jobs.

Reply:
As I understood, see below, the `setup-python` action is taking care of that.

Reply:
Good point, but "caching is turned off by default", see https://github.com/actions/setup-python#caching-packages-dependencies

Reply:
I applied the corresponding changes.